includes/class-freemius.php

```diff
@@ -5481,6 +5563,16 @@ function _require_license_activation_dialog() {
 		}
 
 		/**
+		 * Includes all needed code for the opt-out dialog.
+		 *
+		 * @author Leo Fajardo (@leorw)
+		 * @since  1.2.2
+		 */
+		function _require_optout_dialog() {
```

Comment:
@fajardoleo it seems like it's only used internally in the class so if I'm right, turn it to private and remove the `_` prefix.